web/packages/teleport/src/Discover/ConnectMyComputer/TestConnection/TestConnection.tsx

```diff
@@ -94,36 +94,68 @@ export const TestConnection = (props: AgentStepProps) => {
           {meta.resourceName}
           &rdquo; by starting a session.
         </Text>
-        {refetchNodeAttempt.status === '' ||
-          (refetchNodeAttempt.status === 'processing' && <Indicator />)}
-
-        {refetchNodeAttempt.status === 'error' && (
-          <>
-            <TextIcon mt={2} mb={3}>
-              <Icons.Warning size="medium" ml={1} mr={2} color="error.main" />
-              Encountered Error: {refetchNodeAttempt.statusText}
-            </TextIcon>
-
-            <ButtonPrimary
-              type="button"
-              onClick={() => refetchNode(abortController.current.signal)}
+        {fetchLoginsAttempt.status === '' ||
+          (fetchLoginsAttempt.status === 'processing' && <Indicator />)}
```

Comment:
just double checking that this check was intended or is it:


```suggestion
        {(fetchLoginsAttempt.status === '' ||
          fetchLoginsAttempt.status === 'processing') && <Indicator />}
```

Reply:
Oh, it definitely should be what you suggested. The current version will not show anything if status is `''`. I must have not noticed it because the status is equal to `''` just for a split second.